platform/posix/transport/include/openssl_posix.h

```diff
@@ -120,7 +120,7 @@ typedef struct OpensslCredentials
      * @note By setting this to 0, OpenSSL uses the default value,
      * which is 16384 (SSL3_RT_MAX_PLAIN_LENGTH).
      */
-    size_t maxFragmentLength;
+    uint64_t maxFragmentLength;
```

Comment:
If this fragment Length is limited to 16384 then why is 64bit the most appropriate type here? It seems like this requires only 16 bits?

Reply:
Thanks. Resolved it.